lib/src/intl_suggestors/utils.dart

```diff
@@ -108,7 +108,7 @@ String intlFunctionBody(
 }) {
   var escapedStr = escapeApos(message);
   String delimiter = (isMultiline) ? "'''" : "'";
-  return "Intl.message(${delimiter}${escapedStr}${delimiter}, ${args.isNotEmpty ? 'args: ${args}, ' : ''}name: '$name',)";
+  return "Intl.message(${delimiter}${escapedStr}${delimiter}, ${args.isNotEmpty ? 'args: ${args}, ' : ''}name: '$name')";
```

Comment:
The trailing comma disappears if we regenerate from the AST, which makes lots of test differences, so just don't put it in.